crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -649,6 +649,8 @@ impl OlmMachine {
     /// # Panics
     ///
     /// Panics if a group session for the given room wasn't shared beforehand.
+    ///
+    /// [`share_group_session`]: #method.share_group_session
```

Comment:
Couldn't this be `Self::share_group_session` which is checked to resolve correctly as an intra-doc link, as opposed to this formulation that's just an HTML anchor and thus not checked?

Reply:
maybe. I only did what we do in other instances in this file. Your change also works. fine with, that ...

```suggestion
    /// [`share_group_session`]: Self::share_group_session
```